sites/cnfpt/src/backend/cnfpt/settings.py

```diff
@@ -370,6 +372,38 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
         default="richie", environ_name="RICHIE_ES_INDICES_PREFIX", environ_prefix=None
     )
 
+    # Cache
```

Comment:
All this can be removed as well when Richie is upgraded to a version > beta.9 right? Maybe it will be beneficial to release it before merging this one.

Reply:
Yes, that's the plan !